gammapy/scripts/tests/test_cta_irf.py

```diff
@@ -34,5 +34,16 @@ def test_cta_irf():
     # val = irf.bkg.evaluate(energy=energy, x=offset, y=Quantity(0, 'deg'))
     # assert_quantity_allclose(val, Quantity(247996.974414962, 'm^2'))
 
+
+@requires_dependency('matplotlib')
+@requires_data('gammapy-extra')
+def test_point_like_perf():
```

Comment:
We always try to name tests as similarly to classes / functions as possible.
In this case, my suggestion would be `test_cta_perf`.
Or if you want to mention "point like" for some reason, `test_cta_perf_point_like`.

I'll try this out locally now, to see if I get a useful plot.